themes/survey/fruity_twentythree/css/variations/theme_blueberry.css

```
@@ -15511,122 +15511,323 @@ body .imageselect-listitem label::after {
+  text-align: left;
+}
+.no-tables-general .slider-list .slider-right {
+  text-align: right;
+}
+.dir-rtl .no-tables-general .slider-list .slider-left {
+  text-align: right;
+}
+.dir-rtl .no-tables-general .slider-list .slider-right {
+  text-align: left;
+}
+.no-tables-general table.ls-answers,
```

Biome (2.3.14)

[error] Don't use unknown media feature names.

Unexpected unknown media feature name.
You should use media feature names defined in the CSS Specifications.

(lint/correctness/noUnknownMediaFeatureName)